test/integration/targets/known_hosts/aliases

```diff
@@ -0,0 +1,2 @@
+needs/root
```

Comment:
I don't know!  I was copying what the authorized_key target did, without understanding what these things mean.